client/signup/steps/domains/index.jsx

```diff
@@ -243,6 +244,16 @@ const DomainsStep = React.createClass( {
 			);
 		}
 
+		let fallbackHeaderText,
+			fallbackSubHeaderText;
+		if ( abtest( 'signupDomainsHeadline' ) === 'updated' ) {
+			fallbackHeaderText = this.translate( 'Let\'s find a cool name for your brand new site' ),
+			fallbackSubHeaderText = this.translate( 'Choose a compelling name, it will be the address of your new site!' );
+		} else {
```

Comment:
I prefer this pattern as it makes more sense to me - I assign either value based on a condition.
While
```
let foo = 'bar';
if ( something ) {
  foo = 'lol, nope';
}
```
seems confusing to me - if I can immediately check if a different value should be assigned to this variable, why assign the original value at all?